zoltar-core/src/main/java/com/spotify/zoltar/Predictor.java

```diff
@@ -81,8 +81,8 @@
       final ModelT model,
       final FeatureExtractor<InputT, VectorT> featureExtractor,
       final AsyncPredictFn<ModelT, InputT, VectorT, ValueT> predictFn) {
-    return (input, timeout, scheduler) -> {
-      final List<Vector<InputT, VectorT>> vectors = featureExtractor.extract(input);
+    return (scheduler, timeout, inputs) -> {
```

Comment:
What's the reason for change of order?

Reply:
varargs work best if they are the last arg in a method call

Reply:
ah - duh - that's for varargs, i see now